src/base-element.js

```diff
@@ -954,4 +954,15 @@ export class BaseElement {
   user() {
     return user(this.element);
   }
+
+  /**
+   * Returns this BaseElement instance. This is equivalent to Bento's
+   * imperative API object, since this is where we define the element's custom
+   * APIs.
+   *
+   * @return {!Promise<!./base-element.BaseElement>}
```

Comment:
Seems like the type is not quite correct here? Perhaps we should just do `!Promise<!Object>`?

Reply:
Done.